app/pages/adduser/template.test.js

```diff
@@ -0,0 +1,85 @@
+import { createTestHarness } from '../../test-utils/testHarness';
+import context from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/adduser/template.njk',
+  },
+};
+
+describe('organisations add user page', () => {
+  it('should render a backLink', createTestHarness(setup, (harness) => {
+    harness.request(context, ($) => {
+      const homepageBackLink = $('[data-test-id="go-back-link"]');
```

Comment:
will this actually be going back to homepage? I think just change the name of the const to `backLink`